.github/_test_unit.yaml

```
@@ -0,0 +1,26 @@
+name: Test Unit
+
+on: [push]
+
+jobs:
+  test-unit:
+    runs-on: ubuntu-22.04
+    timeout-minutes: 10
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Use Go
+        uses: actions/setup-go@v5
+        with:
+          go-version: '>=1.22.x'
```

Let's use the Go version from go.mod instead of specifying it manually
```suggestion
          go-version-file: go.mod
```